configuration/steps/commands/base.py

```diff
@@ -20,3 +24,28 @@ def __init__(self, name, workdir: PurePath, user: str = "buildbot"):
     @abstractmethod
     def as_cmd_arg(self) -> list[str]:
         pass
+
+
+class ShellCommandWithURL(steps.ShellCommand):
```

Comment:
Document this please:

This class extend's Buildbot's base ShellCommand, to allow rendering an additional url in the interface. The URL can point to relevant artifacts for developers to use.